src/app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
```

Comment:
Let's improve the server structure. Please create a separate functions for each request and make your server like this:
```suggestion
const server = http.createServer((req, res) => {
if (req.url === '/') {
    handleIndex(req, res);
  }

  if (req.url === '/getJSON' && req.method === 'POST') {
    handleUpload(req, res);
  }
}
```

Reply:
Done